### ISSUE TITLE
Leads' Agenda Items

### ISSUE THREAD

**Issue by hessionra** [ICPI/Coordination#86]
Rolling tab of agenda items for each Monday's Agency Leads' Meeting.

_Monday, August 19th_
- panorama Q3 and Q4 updates
- SI COOP updates

**Comment by hessionra** [comment]
_Monday, August 26th_
- Senior Staff meet with Leads 8/26

**Comment by hessionra** [comment]
_Monday, September 9th_

ICPI SI COOP Proposal
ICPI Leadership Retreat with OGAC
•	Solidifying date with Irum
•	Scheduling ICPI Leads pre-retreat meeting
Engagements
•	Map request - PEPFAR sites that report on lab metrics or are PEPFAR supported labs color-coded differently from all PEPFAR TX sites.
•	ICPI Overview for PEPFAR India Staff
LoE Document

- ICPI Onboarding training October 15 and 16
October Onboarding Registration (link below)

- PVA Onboarding Training Materials
17 modules, possible reduction to 9 modules for ease of completion (per Catherine Nichols)

LINK--https://docs.google.com/forms/d/e/1FAIpQLScZj-imXHzW5pA0vjSh3P13x9ykWp3IFZ7L0mN7psaw-YLZ2g/viewform?usp=sf_link

**Comment by hessionra** [comment]
Monday, September 16th
- [Request for GBV & Treatment Dashboard ](https://github.com/ICPI/Engagements/issues/77)
- ICPI Onboarding Draft Agenda (@jleehurst )
    SAB on 10/16 - do we:
     1.  move dates to 10/17-10/18 (Th/Fr)? 
     2.   move to peace corps? 
- IDS Tentatively Scheduled for 9/24
- Panorama Development
- South Africa TDY/Siyenza/MSTR
- Admin
    - Jasmine on AL
    - Staffing announcements

**Comment by hessionra** [comment]
Monday, September 23

•	ICPI Onboarding Updates 
i.	New Registration Link
ii.	Registrant report (attached)
•	IDS/ATD Status
•	Q3/Q4 Panorama updates
•	Admin
o	Jasmine on AL 9/26-10/8

**Comment by hessionra** [comment]
_Monday, September 30_
@cnichols3h @HoustonJamie 

- Staff updates
- Transition plan
   - Short term plans
   - Longer term plans/priorities
      - Q4 Dossiers
      - ICPI Onboarding training/PVA 2.0
      - SI COOP
- Onboarding Training
   - Attendees (Leads' review)

**Comment by hessionra** [comment]
Monday, October 7th 
- DP Updates
- Engagements
- DIV Updates (Panorama)
- DAQ Updates
- Onboarding Training Updates
     - Final participant list
- LoE document (follow up from last week)
- Transition updates (follow up from last week)

**Comment by HoustonJamie** [comment]
_Monday, September 30_
@cnichols3h @hessionra 

- Discuss any updates re: follow up items from last weeks meeting 
  - Updates from agency leads re: mechanisms for coordinator recruitment 
  - Review the updated LOE format 
  - Discuss comments re: the 3 months priorities slide
  - Confirm with leads that Kimi, Shaylee, Catherine and Jamie are the rotating acting Coordinators (1 month at a time) with specific topic area leads TBD
- DP Updates
- Engagements
  -Update on South Africa OPU request 
  -Update on potential South Africa training request 
  -Update on India rTDY to ICPI in Nov 
- DIV Updates
  -Pano Q4 kick off meeting completed for new dossiers: New dossiers PMTCT, TB/HIV, Prevention, OVC, and Maps
  -Pano Q4 kick off for updates to current dossiers this week: Includes Clinical Cascades, Testing, Treatment, Viral Load and HSS 
- DAQ Updates
   -COP20 Targets now in the MSDs 
   -Data genie user report for Q4 (July 1st - Sept 30th) available 
- Onboarding Training Updates 
   -Final participant list
- Request for new HSS cluster leads starting in 2 weeks (Gordon's last day Oct 14th and Travis on paternity leave Oct 18th-Jan 6th) 
- Update on Spectrum-Datapack-DATIM calls starting on Oct 15h, suggestion for agenda item for leads next Monday Sept 14th.

**Comment by hessionra** [comment]
_Monday, October 21_

@HoustonJamie 

- DP Updates
- DAQ Updates
- Pano Updates
- Engagement Updates
- Admin
- AOB

**Comment by jleehurst** [comment]
Monday, October 28

- Review onboarding feedback presentation @hessionra @HoustonJamie 
- PrEP and Cervical Cancer Dossier development
- Scheduling 1:1s for analysts with Jamie
- Admin
     - Staffing Updates

**Comment by lucasrachel** [comment]
Please add the following to Oct 28 agenda:

PrEP and Cervical Cancer Dossier development

**Comment by annieschwartz** [comment]
Yes, second the Prep and Cervical Cancer Dossier. Can we also talk just about process a bit: how we will do lessons learned from Q4 and and how to feed agenda items to Jamie for the Friday meeting.

**Comment by hessionra** [comment]
/remind me every Friday to make agenda

**Comment by hessionra** [comment]
Leads, _Tuesday November 12th_
